core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -339,6 +346,13 @@ public boolean shutdownWorkerGroupOnStop() {
         return shutdownWorkerGroupOnStop;
     }
 
+    /**
+     * Returns {@link ChannelOption} which use to create {@link ServerBootstrap}.
```

Comment:
which is used?

Reply:
`Returns the {@link ChannelOption}s and their values of {@link Server}'s server sockets.`